src/main/java/de/cyface/collector/handler/FormAttributes.java

```diff
@@ -41,7 +41,27 @@
     /**
      * The operating system version, such as Android 9.0.0 or iOS 11.2.
      */
-    OS_VERSION("osVersion");
+    OS_VERSION("osVersion"),
+    /**
+     * The version of the app that transmitted the measurement.
+     */
+    APPLICATION_VERSION("appVersion"),
+    /**
+     * The length of the measurement in meters.
+     */
+    LENGTH("length"),
+    /**
+     * The count of geo locations in the transmitted measurement.
+     */
+    LOCATION_COUNT("locationCount"),
+    /**
+     * The geo location at the beginning of the track represented by the transmitted measurement.
+     */
+    START_LOCATION("startLocation"),
+    /**
+     * The geo location at the end of the track represented by the transmitted measurement.
+     */
+    END_LOCATION("endLocation");
```

Comment:
What's this for? To be able to load measurements which start/end within a bounding box?

Reply:
I guess the format of the expected string is defined in the API description you created for the shared API right?

Reply:
The end location was requested as a feature. I guess it servers for plausibility checking.

The format is specified at least as an example in the API spec. I would prefer not to split this into three properties, but if necessary we will do so in a future version.